sources/modules/compilation-model/separate-compilation.md

```diff
@@ -0,0 +1,103 @@
+## Module name: Separate Compilation
+
+_Skeleton descriptions are typeset in italic text,_
+_so please don't remove these descriptions when editing the topic._
+
+### Overview
+
+_Provides a short natural language abstract of the module’s contents._
+_Specifies the different levels of teaching._
+
+This module outlines the issues involved in using separate compilation on multiple program files.
+
+------------------------------------------------------------------------
+Level             Objective
+----------------- ------------------------------------------------------
+Foundational:      Awareness of the existence of separate compilation
```

Comment:
Is 'awareness' here the right term, as the ASBATS of the section require the student to also be able to compile/refactor a program into different translation units.

Reply:
+1

Reply:
That was resolved in today's meeting.